packages/ws-server/src/webSocketServer.ts

```diff
@@ -98,6 +111,7 @@ app.ws.use((ctx) => {
     ctx.websocket.on('error', console.error);
 
     ctx.websocket.on('close', function () {
+        connectedClients--;
```

Comment:
Very interesting that it handles increments but not decrements of the connected client count.

Reply:
Ugh, I thought I removed that. It's removed now.  Detailed explanation below.